support-metrics-client/src/test/java/io/confluent/support/metrics/MetricsToKafkaTest.java

```diff
@@ -56,7 +60,7 @@ public void savesAsManyMetricsToFileAsHaveBeenSubmittedBySingleNodeCluster() thr
         "test_metrics");
     String topic = brokerConfiguration.getProperty(KafkaSupportConfig.CONFLUENT_SUPPORT_METRICS_TOPIC_CONFIG);
     int timeoutMs = 10 * 1000;
-    KafkaMetricsToFile kafkaMetricsToFile = new KafkaMetricsToFile(cluster.zookeeperConnectString(), timeoutMs);
+    KafkaMetricsToFile kafkaMetricsToFile = new KafkaMetricsToFile(bootstrapServer(broker.zkClient()));
```

Comment:
Can't we get bootstrap address from `broker` without going through ZK?

Reply:
I tried that first, but the port was somehow `0`. This approach of going via ZK is used elsewhere as well, so I just went with that for now.